.gitignore

```diff
@@ -1,4 +1,5 @@
-/**/target/
+/docker/query-coordinator-assembly.jar
 /.idea/
 /.idea_modules/
 local-query-coordinator.conf
+/**/target/
```

Comment:
Mmm, I'm not entirely sure about that yet.  It seems that the saner use cases are either actually checking in a env file for developer workstation use (under a more descriptive name than `envfile`) or if people want their own customized files for some reason then keeping them under a separate tree might make more sense so they can be safely stored and versioned might make more sense.

The overall question of how to best let people run them locally is still to be worked through.